torch/onnx/symbolic_opset9.py

```diff
@@ -2376,7 +2376,12 @@ def repeat_interleave(g, self, repeats, dim=None, output_size=None):
 
     final_splits = list()
     r_splits = sym_help._repeat_interleave_split_helper(g, repeats, reps, 0)
```

Comment:
Make this change inside `_repeat_interleave_split_helper` so it only has to be done in one place.